lib/spack/spack/cmd/installer/README.md

```diff
@@ -0,0 +1,85 @@
+This README is a guide for creating a Spack installer for Windows using the
```

Comment:
The installer and its related files are AFAIK not a WIP and are used end to end in the Windows CI. The CMake file is driven by Spack's cmd interface, and the README provides more detailed instructions on how to set up the installer, primarily for Spack developers. This README file may want to live somewhere else, I'll leave that call to you, but the CMake is an integral part of creating the installer.
An edition of the installer is created and then tested by the Spack on Windows CI, and then published for use as a build artifact. Users can download that installer and have Spack, Git, and Python installed on their system automatically. We leave these instructions in case a user/developer wants to amend the installer generation process for some reason. We vendor the installer as that is how software is typically installed on Windows. However we leave the getting started info for those who wish to spin up from scratch. I hope this answers your questions.

Reply:
Yes thanks for clarifying: I was mistaken and this should stay.